crates/router/src/connector/globalpay/requests.rs

```diff
@@ -1,268 +1,351 @@
 use serde::{Deserialize, Serialize};
 
-#[derive(Debug, Default, Serialize)]
+#[derive(Debug, Default, Serialize, Clone)]
 pub struct GlobalpayPaymentsRequest {
     /// A meaningful label for the merchant account set by Global Payments.
     pub account_name: String,
     /// The amount to transfer between Payer and Merchant for a SALE or a REFUND. It is always
     /// represented in the lowest denomiation of the related currency.
+    #[serde(skip_serializing_if = "Option::is_none")]
```

Comment:
Whys is this attribute present for all the fields?

Reply:
Request should only have one type of payment method, to skip other payment methods we use skip_serializing_if. With "serde_with" crate instead of using "skip_serializing_if" at every field(if option is none), we can use "skip_serializing_none" for whole struct . Also we only use this at "PaymentMethod" .